crates/proto/README.md

```diff
@@ -2,5 +2,9 @@
 
 This crate contains protobuf bindings for the APIs exposed by the components of the Miden node and domain types.
 
+> ⚠️ **Warning**\
+> This crate is intended for internal use only. For gRPC bindings please see the [miden-node-proto-build](../../proto/README.md) crate.\
+> This crate does not support TLS and cannot be used as a client for the official RPC endpoints.
```

Comment:
github has builtin support for warnings and notes called [alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

```suggestion
> [!WARNING]
> This crate is intended for internal use only. For gRPC bindings please see the [miden-node-proto-build](../../proto/README.md) crate.\
> This crate does not support TLS and cannot be used as a client for the official RPC endpoints.
```